### PR TITLE
alttab: `doCheck`+`gpl3Plus`

### PR DESCRIPTION
**Pull request by sgraf812**
###### Motivation for this change

`alttab` has a `make check` target since 1.6.0. This PR enables running the target as part of CI, via `doCheck = true;`.

I also clarified the license to `gpl3Plus`, as the license header in e.g. https://github.com/sagb/alttab/blob/1c0754e6d2da346a08b06725ba716350553edf48/src/alttab.c clearly says "any later".

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by dotlambda — CHANGES_REQUESTED** [review]
Tests fail.

**Comment by SuperSandro2000** [comment]
> Tests fail.

They don't fail per say but execute nothing according to the logs of ofborg.

**Comment by dotlambda** [comment]
> They don't fail per say but execute nothing according to the logs of ofborg.

They fail if you merge this with master (where alttab was updated 1.6.0).

**Comment by sgraf812** [comment]
Urgh, yes. I'll rebase and try again!

File: pkgs/tools/X11/alttab/default.nix [hunk 1]
```diff
@@ -35,10 +35,22 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  doCheck = true;
+
+  checkInputs = [
+    coreutils
+    procps
+    python38Packages.xvfbwrapper
```
Comment (dotlambda) [inline pkgs/tools/X11/alttab/default.nix:43]:
```suggestion
    python3Packages.xvfbwrapper
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Pull request merged**